### PR TITLE
e2e: Now works on travis.

### PR DESCRIPTION
**Pull request by ashcrow**
Requires: https://github.com/projectatomic/commissaire-service/pull/49

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
   - "3.5"
 install:
   - "pip install tox"
 script:
-  - "tox -v -e py35"
+   - "wget https://github.com/coreos/etcd/releases/download/v3.1.0/etcd-v3.1.0-linux-amd64.tar.gz"
+   - "tar xvzf etcd-v3.1.0-linux-amd64.tar.gz"
+   - "pushd etcd-v3.1.0-linux-amd64"
+   - "./etcd &"
+   - "popd"
```
Comment (mbarnes) [inline .travis.yml]: Just to mention, there's a backported `etcd` package for Ubuntu Trusty in this PPA:
https://launchpad.net/~cloud-green/+archive/ubuntu/trusty-backports

It's an older version, but I'm not aware of any minimum requirement for Commissaire.

Could maybe simplify `etcd` installation with this method:
https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-from-a-custom-APT-repository

**Review by mbarnes — APPROVED** [review]
Optional suggestion, but this looks good either way.  Nice!

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+  - redis-server
 python:
   - "3.5"
+before_install:
+  - "sudo apt-get install etcd -y"
 install:
   - "pip install tox"
 script:
-  - "tox -v -e py35"
+  - "etcd &"
```
Comment (mbarnes) [inline .travis.yml]: Highly nitpicky, but if you're still tweaking this you could `sudo systemctl start etcd` in `before_install:`.

Not a big deal though.

File: .travis.yml [hunk 3]
```diff
@@ -1,12 +1,18 @@
 language: python
+services:
+  - redis-server
 python:
   - "3.5"
+before_install:
+  - "sudo add-apt-repository ppa:cloud-green/trusty-backports"
```
Comment (mbarnes) [inline .travis.yml]: Tack on `-y` to avoid the interactive prompt.

**Comment by mbarnes** [comment]
Sorry... my suggestion turned out to be more trouble than it was worth.  :disappointed:

**Comment by ashcrow** [comment]
@mbarnes the PPA packages are busted. I'm going to revert back to wget :frowning_face:

**Comment by mbarnes** [comment]
> @mbarnes the PPA packages are busted. I'm going to revert back to wget :frowning_face:

Understood.

**Comment by ashcrow** [comment]
:arrow_up:

**Review by mbarnes — APPROVED** [review]
Didn't realize the Travis CI environment was so ancient.  I'll accept whatever works at this point.  Apologies for the diversion.

**Comment by ashcrow** [comment]
[(3) [multi-host-mgr] Add bdd back to travis](https://trello.com/c/hNVzXU8b/664-3-multi-host-mgr-add-bdd-back-to-travis)

**Pull request merged**